src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffendersResourceTransferImpTest.kt

```diff
@@ -54,6 +62,13 @@ class OffendersResourceTransferImpTest : ResourceTest() {
   @Autowired
   private lateinit var dataLoader: DataLoaderRepository
 
+  @MockBean
+  private lateinit var workflowTaskService: WorkflowTaskService
+
+  private val team: Team by lazy {
+    dataLoader.load(TeamBuilder())
+  }
+
```

Comment:
we only need one team that is static created rather then one per test since we never mutate the team